src/figuresClasses.ts

```diff
@@ -1,11 +1,94 @@
-export interface Figure {}
+type Shape = `triangle` | `circle` | `rectangle`;
+type Color = `red` | `green` | `blue`;
+// enum Parameters {
+//   a: number,
+//   b: number,
+//   c: number,
+//   radius: number,
+//   color: color,
+// }
 
-export class Triangle implements Figure {}
+export interface Figure {
+  shape: Shape;
+  color: Color;
+  getArea: () => number;
+}
+
+function longestSide(a: number, b: number, c: number): boolean {
```

Comment:
1. From the function name does not clear what it is doing. As of now, it sounds like it just picks and returns the longest side.
2. This function should be a static method inside `Triangle` class